app/js/shipyard/factory-ship.js

```diff
@@ -548,7 +548,12 @@ angular.module('shipyard').factory('Ship', ['Components', 'calcShieldStrength',
   };
 
   Ship.prototype.useUtility = function(group, rating, clobber) {
-    var component = Components.findHardpoint(group, 0, rating);
+    var component;
+    if (group == 'cm') {
+      component = Components.findHardpoint(group, 0, null, rating); //overload Rating as Name for countermeasures
```

Comment:
Rather than overload the rating parameter, I think it would be better to add another parameter `name`

(This would have to be done for `useUtility` in `controller-outfit.js`)

The `clobber` parameter isn't used anywhere at the moment, so it can be left as the last argument